From 34a0c0a567cdc8aff47d946776eca1bf61c4d68d Mon Sep 17 00:00:00 2001
From: "Chen, Yu" <yu.y.chen@intel.com>
Date: Thu, 16 May 2019 17:22:58 +0800
Subject: [PATCH 3/8] Don't build the libpolicy-subsystem in aosp

because there's same module provided in bsp side

Change-Id: Ibb2090568f67b3419fd5a05c8f1f0331e91de17f
Tracked-On:
Signed-off-by: Chen, Yu <yu.y.chen@intel.com>
---
 .../parameter-framework/plugin/Android.bp     | 36 -------------------
 1 file changed, 36 deletions(-)

diff --git a/services/audiopolicy/engineconfigurable/parameter-framework/plugin/Android.bp b/services/audiopolicy/engineconfigurable/parameter-framework/plugin/Android.bp
index 2685c6dc8c..e69de29bb2 100644
--- a/services/audiopolicy/engineconfigurable/parameter-framework/plugin/Android.bp
+++ b/services/audiopolicy/engineconfigurable/parameter-framework/plugin/Android.bp
@@ -1,36 +0,0 @@
-cc_library_shared {
-    name: "libpolicy-subsystem",
-    srcs: [
-        "PolicySubsystemBuilder.cpp",
-        "PolicySubsystem.cpp",
-        "InputSource.cpp",
-        "Stream.cpp",
-        "ProductStrategy.cpp",
-    ],
-    cflags: [
-        "-Wall",
-        "-Werror",
-        "-Wextra",
-        "-fvisibility-inlines-hidden",
-        "-fvisibility=hidden",
-    ],
-    header_libs: [
-        "libbase_headers",
-        "libaudiopolicycommon",
-        "libaudioclient_headers",
-        "libaudiopolicyengine_interface_headers",
-        "libaudiopolicyengineconfigurable_interface_headers",
-    ],
-    static_libs: [
-        "libaudiopolicycomponents",
-        "libaudiopolicyengine_common",
-        "libpfw_utility",
-    ],
-    shared_libs: [
-        "libaudiopolicyengineconfigurable",
-        "liblog",
-        "libutils",
-        "libmedia_helper",
-        "libparameter"
-    ],
-}
--
2.21.0
